pkg/config/config.go

```diff
@@ -24,7 +24,7 @@ const (
 )
 
 var (
-	cfg     *GochanConfig
+	Cfg     *GochanConfig
```

Comment:
Cfg should be private here, to avoid being accidentally overwritten by other packages. The relevant sections can be accessed via the following functions:
- `GetSQLConfig()` for SQL related things (though in most cases, the public functions in the `gcsql` package should make this unnecessary outside of the `gcsql` package), `
- `GetSystemCriticalConfig()` for things like directories (`TemplateDirectory`, `LogDirectory`, etc), the address and port gochan is listening on (`ListenIP` and `Port` respectively), etc
- `GetSiteConfig()` for things like the site name and slogan, minification info, etc
- `GetBoardConfig(board)` for board-specific configuration, or for the global board/post/upload configuration if an empty string is passed as the board
- `UpdateBoardConfig(board)` for reloading /path/to/docroot/<board>/board.json

Reply:
Hey, 
yup i know for that but i just do that to make tests works will waiting your review.
I'll try to fix that with your advice thanks :)